objects/snake.cpp

```diff
@@ -174,63 +217,84 @@ class Snake
         for (int indexBody = 1; indexBody < size; indexBody++)
         {
             setBodyMoveSide(indexBody);
+            isBack = false;
```

Comment:
Question:
How about 
```suggestion
            isBack = (indexBody == size - 1);
```